python/katana/local/_graph.pyx

```diff
@@ -94,13 +126,13 @@ cdef class GraphBase:
         """
         return self.topology().num_edges()
 
-    def loaded_node_schema(self):
+    def loaded_node_schema(self) -> pyarrow.Schema:
         """
         Return the `pyarrow` schema for the node properties loaded for this graph.
         """
         return pyarrow_wrap_schema(self.underlying_property_graph().loaded_node_schema())
 
-    def loaded_edge_schema(self):
+    def loaded_edge_schema(self) -> pyarrow.Schema:
```

Comment:
Not really relevant to this PR, but I think we might not want to be returning pyarrow objects at all in the API. Pyarrow is not a library data scientists are at all familiar with, and having to look up the arrow API to understand how to use what our API returns is a real speed bump when reading the docs for the first time.

Reply:
Yea. I agree. We should phase it out. This PR is not the last step in that process at all.